cvs diff (in directory C:\Development\wxWindows\src\univ\)
cvs server: Diffing .
Index: slider.cpp
===================================================================
RCS file: /home/wxcvs/wxWindows/src/univ/slider.cpp,v
retrieving revision 1.7
diff -r1.7 slider.cpp
790c790,794
<     if ( !rectShaft.Inside(pt) )
---
>     wxRect rectThumb;
>     CalcThumbRect(&rectShaft, &rectThumb, NULL);
> 
> 	// check for possible shaft or thumb hit
>     if (!rectShaft.Inside(pt) && !rectThumb.Inside(pt))
795,797d798
<     // inside the shaft, where is it relatively to the thumb?
<     wxRect rectThumb;
<     CalcThumbRect(&rectShaft, &rectThumb, NULL);

 	  	 
